mentat/terminal/output.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any
+
+from prompt_toolkit import print_formatted_text
+from prompt_toolkit.formatted_text import FormattedText
+
+from mentat.session_stream import StreamMessage
+
+
+def _print_stream_message_string(
+    content: Any,
+    end: str = "\n",
+    color: str | None = None,
+    flush: bool = False,
+):
+    if color:
+        f_color = color.replace("_", "").replace("light", "bright")
+        if f_color != "" and not f_color.startswith("ansi"):
+            f_color = "ansi" + f_color
+        print_formatted_text(FormattedText([(f_color, content)]), end=end, flush=flush)
+    else:
+        print(content, end=end, flush=True)
```

Comment:
Builtin `print` properly renders the color codes added by termcolor's `colored`, wasn't able to get the rendering working for `print_formatted_text`. Once we move away from using termcolor on the core library this code will be updated.